protected/humhub/modules/content/controllers/ContentController.php

```diff
@@ -48,6 +52,61 @@ public function behaviors()
         ];
     }
 
+    /**
+     * @param $fileList array pre-uploaded File GUIDs to be attached to the new content
+     * @return string
+     */
+    public function actionCreate()
```

Comment:
Maybe we could separate this code into a new. `ShareIntendController` - 
This could also be used by the PWA later.

Reply:
The idea here was to later also allow creating content from the Dashboard: https://github.com/humhub/humhub-internal/issues/526 and public https://github.com/humhub/humhub/issues/5396

That's why I have created this action not only to share files. The `$fileList` param is optional.

This way, we would have the same code for the modal to create a content from the dashboard.

The PWA could also use it. It can be open by JS with this code:

```js
$('#globalModal').modal('show');
$('#globalModal .modal-content').load(contentCreateUrlAndParams);
```

What do you think?

Reply:
Hmm, I think I would prefer to deal with them separately for now.
At the moment it is not yet clear how we would like to implement this with the “Dashboard Content”.